public/pages/CreateMonitor/containers/DefineMonitor/DefineMonitor.js

```diff
@@ -220,35 +259,51 @@ class DefineMonitor extends Component {
     const { dataTypes, response, performanceResponse } = this.state;
     const { index, searchType, timeField } = values;
     const isGraph = searchType === SEARCH_TYPE.GRAPH;
+    const isHTTP = searchType === SEARCH_TYPE.HTTP;
```

Comment:
isHTTP -> isHttp

Reply:
Same for other places below, use Http instead of HTTP

Reply:
Got it, will fix these.